modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8_{context}"]
+= {mtc-full} 1.8 release notes
+
+[id="new-features-and-enhancements-1-8{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+This release has no new features and enhancements.
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
+This release has no deprecated features.
+
+[id="resolved-issues-1-8_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.Indirect migration is stuck on backup stage
+
+In previous releases, indirect migration becomes stuck at the backup stage, due to invalid image name error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2233097[*BZ#2233097*])
+
+.PodVolumeRestore remain In Progress keeping the migration stuck at Stage Restore
+
+In previous releases, on performing any indirect migration, the migration becomes stuck at the `Stage Restore` step, waiting for the `podvolumerestore` to be completed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233868[*BZ#2233868*])
+
```

Comment:
PodVolumeRestore remains... 
"any indirect migration" or "an indirect migration"?
Please double-check that `Stage Restore` is two words. Likewise "In Progress"  -- if that is not an actual status or stage name, write it in lower case. You might want to use "in progress" in the actual description, too.